docs/zephyr-integration.md

```diff
@@ -0,0 +1,493 @@
+    "priorityName": "High"
+  }
+}
+```
+
+#### `zephyr_update_test_case`
+
+Update existing test case properties.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `updateData` (required): Object containing fields to update
+  - `name` (optional): Updated test case name
+  - `objective` (optional): Updated test case objective
+  - `precondition` (optional): Updated test precondition
+  - `estimatedTime` (optional): Updated estimated time in minutes
+  - `componentId` (optional): Updated component ID
+  - `priorityName` (optional): Updated priority name
+  - `statusName` (optional): Updated status name
+  - `folderId` (optional): Updated folder ID
+  - `ownerId` (optional): Updated owner ID
+  - `labels` (optional): Updated test case labels
+  - `customFields` (optional): Updated custom fields
+
+#### `zephyr_add_test_script`
+
+Add executable script or detailed instructions to test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `text` (required): Test script content
+- `type` (optional): Test script type - must be 'plain' or 'bdd'. Defaults to 'plain'.
+
+#### `zephyr_add_test_steps`
+
+Add structured test steps with expected results.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `testSteps` (required): Array of test steps with descriptions and expected results
+  - `description` (required): Step description
+  - `expectedResult` (required): Expected result
+  - `testData` (optional): Test data for this step
+
+**Example:**
+```json
+{
+  "testCaseKey": "PROJ-T123",
+  "testSteps": [
+    {
+      "description": "Navigate to login page",
+      "expectedResult": "Login form is displayed",
+      "testData": "URL: /login"
+    },
+    {
+      "description": "Enter valid credentials",
+      "expectedResult": "User is authenticated",
+      "testData": "user: admin, pass: secure123"
+    }
+  ]
+}
+```
+
+#### `zephyr_link_test_case_to_issue`
+
+Create traceability link between test case and Jira issue.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `issueKey` (required): JIRA issue key (e.g., PROJECT-123)
+
+### Test Planning
+
+Organize test activities with test plans.
+
+#### `zephyr_list_test_plans`
+
+List test plans for a project.
+
+**Parameters:**
+- `projectKey` (required): JIRA project key
+
+### Test Case Links and Versions
+
+Manage test case relationships and versioning.
+
+#### `zephyr_get_test_case_links`
+
+Get all links for a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+
+#### `zephyr_create_test_case_web_link`
+
+Create a web link for a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `webLinkData` (required): Web link data with URL and optional description
```

Comment:
do we really need to have webLinkData? I understand that webLinkData would include url and description, but from here it seems to me different things and I do not think we need to have it at all..